build/commands/lib/config.js

```diff
@@ -409,7 +409,7 @@ Config.prototype.buildArgs = function () {
     // Component builds are not supported for iOS:
     // https://chromium.googlesource.com/chromium/src/+/master/docs/component_build.md
     args.is_component_build = false
-    args.ios_deployment_target = '12.0'
+    args.ios_deployment_target = '13.0'
```

Comment:
we should remove this line. We don't want to override the default value here

Reply:
Removed in c7df32f9ab0edf123f6b8f6faad6d1ada8b10882